manpages/lumina-screenshot.8

```diff
@@ -0,0 +1,18 @@
+.Dd October 20, 2017
+.Dt LUMINA-SCREENSHOT 8
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-screenshot
+.Nd ("PrintScreen" button by default) creates, modifies, and saves screenshots.
```

Comment:
The ("PrintScreen" button by default) phrase is better in the full .Sh DESCRIPTION section.